test/spec/modules/eids_spec.js

```diff
@@ -417,6 +417,39 @@ describe('eids array generation for known sub-modules', function() {
     });
   });
 
+  it('pubcid', function() {
+    const userId = {
+      pubcid: {'id': 'sample_id'}
```

Comment:
We should also test that the flat `pubcid` value still works. This is the case where `pubcid=sample_id`.